pkg/serving/v1/client_mock.go

```diff
@@ -202,6 +202,28 @@ func (c *MockKnServingClient) GetConfiguration(name string) (*servingv1.Configur
 	return call.Result[0].(*servingv1.Configuration), mock.ErrorOrNil(call.Result[1])
 }
 
+// CreateRevision records a call CreateRevision
```

Comment:
Could you please also add a call to the recorder and the mock methods to `client_mock_test` ? This keeps the test coverage happy :)

Reply:
Sure, I've missed that.